taichi/backends/metal/shaders/runtime_structs.metal.h

```diff
@@ -102,6 +128,21 @@ STR(
 
       Extractor extractors[kTaichiMaxNumIndices];
     };
+
+    struct ElementCoords { int32_t at[kTaichiMaxNumIndices]; };
+
+    struct ListgenElement {
+      ElementCoords coords;
+      // Memory offset from a given address.
+      // * If in_root_buffer() is true, this is from the root buffer address.
+      // * O.W. this is from the |id|-th NodeManager's |elem_idx|-th element.
```

Comment:
```suggestion
      // * O/W this is from the |id|-th NodeManager's |elem_idx|-th element.
```

It seems that I can only point out trivial English usage issues in this PR - took me a while to realize `O.W.` meant `otherwise`. `O/W` may be the more common used form?

Reply:
ha, sorry about that...